bin/faucet/src/api/get_metadata.rs

```diff
@@ -21,11 +23,12 @@ impl Serialize for Metadata {
         S: Serializer,
     {
         use serde::ser::SerializeStruct;
-        let mut state = serializer.serialize_struct("Metadata", 4)?;
-        state.serialize_field("id", &self.id.to_bech32())?;
+        let mut state = serializer.serialize_struct("Metadata", 5)?;
+        state.serialize_field("id", &self.id.inner().to_hex())?;
         state.serialize_field("issuance", &self.issuance.read().base_units())?;
         state.serialize_field("max_supply", &self.max_supply.base_units())?;
         state.serialize_field("decimals", &self.decimals)?;
+        state.serialize_field("explorer_url", &self.explorer_url)?;
```

Comment:
nit: This and the account ID would be static fields, so I wonder if there is a way to somehow embed them in the HTML/JS responses instead of getting them dynamically.